### DISCUSSION TITLE
欢迎大家一起交流分享

### DISCUSSION THREAD

**Discussion by Chanzi-keji** [Chanzi-keji/chanzi#9] [General]
大家代码审计中的各种问题，欢迎交流探讨

**Comment by pykiller** [comment]
老哥规则导入，需要什么后缀啊cypher 好像不太行

**Reply by Chanzi-keji** [reply]
需要把规则放入文件夹，然后加载时选择文件夹。您参考一下Wiki自定义规则页面

**Comment by qianniaoge** [comment]
我想要在内部网络环境使用，不进行互联网连通，这个时候需要秘钥来认证登录，而不是微信登录。

**Reply by Chanzi-keji** [reply]
> 我想要在内部网络环境使用，不进行互联网连通，这个时候需要秘钥来认证登录，而不是微信登录。

离线使用这个需求暂时还没有计划支持，这里边有几个原因，1.从长期看我们还是希望保持简洁的使用风格，所以采用微信扫码登录支付的方式相比硬件绑定license等方式是最方便的 2 我们的一些功能也需要联网才有更好的使用体验，比如代码导航需要下载Maven gradle依赖的jar包  3.未来考虑的一些功能也还是有联网需求，比如ai自动审计等也会需要联网能力。

**Reply by Chanzi-keji** [reply]
> 我想要在内部网络环境使用，不进行互联网连通，这个时候需要秘钥来认证登录，而不是微信登录。

2025.3.3 版本支持了离线使用，登录需要联网，登录后就可以无网使用了

**Comment by Stardustin** [comment]
2.0之后的版本,用着用着就会卡死(程序未响应),目前还是只能用2.0

**Reply by Chanzi-keji** [reply]
> 2.0之后的版本,用着用着就会卡死(程序未响应),目前还是只能用2.0

是2025.2.0之后的版本吗？是在做什么操作时卡死呢？

**Reply by Stardustin** [reply]
是的,只要打开任一任务,点进数据流,或者ctrl+左键都很容易卡死,当然不止这两个操作,挺多操作都容易卡死的.但目前2025.2.0是没这个问题

**Reply by Chanzi-keji** [reply]
> 是的,只要打开任一任务,点进数据流,或者ctrl+左键都很容易卡死,当然不止这两个操作,挺多操作都容易卡死的.但目前2025.2.0是没这个问题

好的 ，您是mac还是Windows呢？我们分析一下原因

**Reply by Stardustin** [reply]
windows

**Reply by Chanzi-keji** [reply]
> 是的,只要打开任一任务,点进数据流,或者ctrl+左键都很容易卡死,当然不止这两个操作,挺多操作都容易卡死的.但目前2025.2.0是没这个问题

卡死之后等待一会还能恢复吗？还是只能kill进程关闭？

**Reply by Stardustin** [reply]
每次卡死都无法恢复

**Reply by Chanzi-keji** [reply]
> 每次卡死都无法恢复

好的，了解了，非常感谢您的反馈

**Reply by Chanzi-keji** [reply]
> 2.0之后的版本,用着用着就会卡死(程序未响应),目前还是只能用2.0

您试试新版本能否解决您的问题

**Reply by Chanzi-keji** [reply]
> > 2.0之后的版本,用着用着就会卡死(程序未响应),目前还是只能用2.0
> 
> 您试试新版本能否解决您的问题

2025.3.1 版本

**Reply by Stardustin** [reply]
3.1版本解决了,很棒

**Reply by Chanzi-keji** [reply]
> 3.1版本解决了,很棒

感谢您的反馈与支持
